uavcan/src/session/std_vec.rs

```diff
@@ -72,12 +72,11 @@ where
         // TODO maybe some of the logic here can be skipped with anon transfers.
         let session = frame.source_node_id.unwrap();
         // Create default session if it doesn't exist
-        if !self.sessions.contains_key(&session) {
+        if let std::collections::hash_map::Entry::Vacant(e) = self.sessions.entry(session) {
```

Comment:
In my implementation of heapbased sessions in Alex/no_sdt I rewrote the update function. It can be used here as well.